cmd/detectExecuteScan_test.go

```diff
@@ -69,15 +72,19 @@ func TestAddDetectArgs(t *testing.T) {
 				ProjectName:    "testName",
 				ProjectVersion: "1.0",
 				CodeLocation:   "testLocation",
+				FailOn:         "BLOCKER,CRITICAL,MAJOR",
 				Scanners:       []string{"source"},
 				ScanPaths:      []string{"path1", "path2"},
+				Groups:         "testGroup",
 			},
 			expected: []string{
 				"--testProp1=1",
 				"--blackduck.url=https://server.url",
 				"--blackduck.api.token=apiToken",
 				"--detect.project.name=testName",
 				"--detect.project.version.name=1.0",
+				"--detect.policy.check.fail.on.severities=BLOCKER,CRITICAL,MAJOR",
+				"--detect.project.user.groups=testGroup",
```

Comment:
please test also with multiple groups

Reply:
I have added another test case to check multiple groups. And tested the generated CLI with a scan